### ISSUE TITLE
Dialyzer errors

### ISSUE THREAD

**Issue by pedro-gutierrez** [Gravity-Core/graphism#124]
When creating a new app with [graphism_new](https://github.com/pedro-gutierrez/graphism_new), the following dialyzer errors are produced out of the box:

```
Total errors: 3, Skipped: 0, Unnecessary Skips: 0
done in 0m1.28s
lib/***/schema.ex:1:pattern_match
The pattern can never match the type.

Pattern:
false

Type:
true

________________________________________________________________________________
lib/***/schema.ex:1:pattern_match_cov
The pattern
variable_

can never match, because previous clauses completely cover the type
{_, nil | [any()]}.

________________________________________________________________________________
lib/***/schema.ex:1:pattern_match
The pattern can never match the type.

Pattern:
{:ok, :belongs_to, _, _, _}

Type:
{:error, :unknown_field}

________________________________________________________________________________
done (warnings were emitted)
Halting VM with exit status 2
```

**Comment by pedro-gutierrez** [comment]
The error on the `{:ok, :belongs_to, _, _, _}` happens on entities that have no relations.
The implementation of `sort_column/1` in the api module needs to omit the check on `{:ok, :belongs_to, _, _, _}` since this pattern is never returned.

**Comment by pedro-gutierrez** [comment]
Fixed in v0.8.2
